etc/submit.sh

```diff
@@ -1,4 +1,13 @@
 #!/bin/bash
+# should be a bit nicer than before
+echo "======== WMAgent bootstrap STARTING at $(TZ=GMT date) on $(hostname -f) ========"
+echo "User id : $(id)"
+echo "Local time : $(date)"
+echo "Current system : $(uname -a)"
+echo "Arguments are $@"
+echo "Current hostname: $(hostname -f)"
```

Comment:
Actually, the hostname is already in line 3. So you can remove this line 8 and update line 3 with `at $GLIDEIN_CMSSite` as well.